src/components/page-configuration/delete-application-form.js

```diff
@@ -77,7 +78,12 @@ export class DeleteApplicationForm extends React.Component {
                   </Button>
                 </div>
                 <div className="accordion__content">
-                  <Typography>This action can not be undone.</Typography>
+                  <Alert type="danger" className="icon">
```

Comment:
The color contrast makes it hard to read the text. Would it be better to use <Alert type="warning"> and black text?